File: common_library/src/main/java/com/phone/common_library/bean/AddressBean.java
package com.phone.common_library.bean;

public class AddressBean implements Cloneable {

    private String county;
    private String city;

    public AddressBean() {
    }

    public AddressBean(String county, String city) {
        this.county = county;
        this.city = city;
    }

    public String getCounty() {
        return county;
    }

    public void setCounty(String county) {
        this.county = county;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    @Override
    protected AddressBean clone() throws CloneNotSupportedException {
        return (AddressBean) super.clone();
    }
}

File: library_greendao/src/main/java/com/phone/library_greendao/bean/AddressBean.kt
package com.phone.library_greendao.bean

class AddressBean(county: String, city: String) : Cloneable {

    var county: String? = county
    var city: String? = city

    @Throws(CloneNotSupportedException::class)
    public override fun clone(): AddressBean {
        return super.clone() as AddressBean
    }

    override fun toString(): String {
        return "AddressBean{" +
                "county='" + county + '\'' +
                ", city='" + city + '\'' +
                '}'
    }
}